en/reference/query-language-reference.html

```diff
@@ -657,7 +675,7 @@ <h2 id="where">where</h2>
         <td>Singlevalue or <a href="../schemas.html#multivalue-field">multivalue</a>
         <a href="../schemas.html#indexing">attribute or index field</a>
         with basic type <a href="schema-reference.html#byte">byte</a>, <a href="schema-reference.html#int">int</a>,
-        <a href="schema-reference.html#long">long</a> or <a href="schema-reference.html#string">string</a>.
+        <a href="schema-reference.html#long">long</a> or <a href="schema-reference.html#string">string</a>. String fields must have <code>match:word</code> or <code>match:text</code>.
```

Comment:
"_or <code>match:text</code>_" ? Shouldn't we avoid match:text when using the in operator?

Reply:
Thanks!